src/index.html

```diff
@@ -9,6 +9,93 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="nav__div">
+        
+        <img
+         class="image"
+         href="home"
+         src="./images/logo.png"
+         alt="moyo"
+        >
+      </div>
+      
+      <nav class="nav__lint">
+        <ul class="nav__item">
```

Comment:
```suggestion
        <ul class="nav__list">
```
it's not an item, it's list